app/routes/articles/index.tsx

```diff
@@ -10,44 +8,18 @@ const ArticleRoute = () => {
   const { path } = useRouteMatch();
 
   return (
-    <UserContext.Consumer>
-      {({ currentUser, loggedIn }) => (
-        <Switch>
-          <Route exact path={path} component={Overview} />
-          <RouteWrapper
-            exact
-            path={`${path}/new`}
-            passedProps={{
-              currentUser,
-              loggedIn,
-            }}
-            Component={ArticleEditor}
-          />
-          <RouteWrapper
-            exact
-            path={`${path}/:articleIdOrSlug`}
-            passedProps={{
-              currentUser,
-              loggedIn,
-            }}
-            Component={ArticleDetail}
-          />
-          <RouteWrapper
-            exact
-            path={`${path}/:articleId/edit`}
-            passedProps={{
-              currentUser,
-              loggedIn,
-            }}
-            Component={ArticleEditor}
-          />
-          <Route component={PageNotFound} />
-        </Switch>
-      )}
-    </UserContext.Consumer>
+    <Switch>
+      <Route exact path={path} component={Overview} />
+      <Route exact path={`${path}/new`} component={ArticleEditor} />
+      <Route
+        exact
+        path={`${path}/:articleIdOrSlug`}
+        component={ArticleDetail}
+      />
+      <Route exact path={`${path}/:articleId/edit`} component={ArticleEditor} />
+      <Route component={PageNotFound} />
+    </Switch>
```

Comment:
This looks so much better!
I assume that it's on purpose that only the Article Editor is login Guarded